file-formats/edcc/examples/z_aff_example_edcc.edcc.json

```
@@ -0,0 +1,144 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Sample Consistency Scenario",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "cloudDevelopment"
+  },
+  "country": "en",
+  "taxAuthorityMessages": [
+    {
+      "messageType": "EXAMPLE_MESSAGE_TYPE",
+      "taxAuthorityDocumentType": "EXAMPLE_INVOICE"
+    }
+  ],
+  "taxAuthorityTables": [
+    {
+      "tableName": "EXAMPLE_TAXAUTHORITY_TABLES"
+    }
+  ],
+  "comparisonTypes": [
+    {
+      "comparisonType": "TaxAuthV/sEdoc",
+      "description": "Tax Authority V/s Edocument",
+      "positionNumber": "1",
+      "className": "CL_EDC_EXAMPLE_VALTYPE",
+      "dataSource": "CDS_COMPARISON_EXAMPLE",
+      "existenceCheck": {
+        "checkId": "EXISTENCE_CHECK",
+        "fieldName": "CDS_COMPARISON_EXAMPLE-FIELD1",
+        "fieldValue": "X",
+        "resultProcess": "EXMISMATCH"
+      },
+      "statusCheck": [
```

This is an array, does it make sense to have it in plural?
```suggestion
      "statusChecks": [
```